bin/proving-service/src/commands/worker.rs

```diff
@@ -65,6 +87,9 @@ pub struct StartWorker {
     /// The port of the worker
     #[clap(short, long, default_value = "50051")]
     port: u16,
+    /// The port of the HTTP server for the information endpoint
+    #[clap(short, long, default_value = "3000")]
+    info_port: u16,
     /// The type of prover that the worker will be
     #[clap(flatten)]
     prover_type: ProverTypeSupport,
```

Comment:
Related to one of the previous comments: should we not restrict each worker to a single type? I'm not sure I see situations under which we'd need a single worker to handle heterogeneous proofs.